cypress/integration/unit_tests/create_score_asset_test.js

```diff
@@ -162,6 +162,25 @@ describe('Unit tests for create_score_asset.js', () => {
         });
   });
 
+  it('handles no svi/income assets', () => {
+    testData.asset_data.income_asset_paths = {};
+    // Won't actually get null for SVI path dict, but fine to tolerate it.
+    testData.asset_data.svi_asset_paths = null;
+    const promise = createScoreAsset(testData);
+    expect(promise).to.not.be.null;
+    cy.wrap(promise)
+        .then(() => {
+          expect(exportStub).to.be.calledOnce;
+          return convertEeObjectToPromise(exportStub.firstCall.args[0]);
+        })
+        .then((result) => {
+          const features = result.features;
+          expect(features).to.have.length(1);
+          expect(features[0]['properties']['MEDIAN INCOME']).to.be.undefined;
```

Comment:
check that other things are defined?

Reply:
Nothing else is defined, because this feature doesn't have any properties (besides score) set at all, because it wasn't created by the score code, just created manually. I think it's fine to leave as is: if we get stricter about requiring other fields we can add them to the test then.